app/models/quarter.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class Quarter < ApplicationRecord
+  belongs_to :school_year
+  has_many :classrooms, dependent: :destroy
+
+  validates :number,
+            presence: true,
+            inclusion: { in: 1..4 },
+            uniqueness: { scope: :school_year_id }
+
+  scope :ordered, -> { order(:number) }
+
+  def next
+    school_year.quarters.ordered.find_by(number: number + 1)
+  end
+
+  def previous
+    school_year.quarters.ordered.find_by(number: number - 1)
+  end
```

Comment:
I'm going to revisit this in future work, this is more experimental at the moment.

Reply:
I think this will work for a V1 anyway.  I don't think we need to worry about a previous quarter from a previous year but I'll add it to my list to check on.